docs/reference/migration/migrate_5_4.asciidoc

```diff
@@ -0,0 +1,8 @@
+[[breaking-changes-5.4]]
+== Breaking changes in 5.4
+
+[[breaking_54_delete_by_query_changes]]
```

Comment:
I don't think we need a separate section for DBQ, maybe "REST API changes" would be better?